### DISCUSSION TITLE
Uninstall SEB 2.4

### DISCUSSION THREAD

**Discussion by sjbmol-lw** [SafeExamBrowser/seb-win-refactoring#464] [Q&A]
Hi

I'm trying to uninstall SEB 2.4 but windows won't let me. The documentation says to find the exact version that is installed. But 2.4 is not available and 2.4.1 does not work.

How would I uninstall this version of SEB 2.4?

Thank you in advance.

**Comment by dbuechel** [comment]
> I'm trying to uninstall SEB 2.4 but windows won't let me

Could you please elaborate a bit about what error message or alike Windows is showing? Normally, you should be able to uninstall SEB directly within the list of installed applications in the Windows control panel.

**Reply by sjbmol-lw** [reply]
When trying to delete with the windows control panel I get the following message : 
![Schermafbeelding 2022-09-15 082925](https://user-images.githubusercontent.com/107043289/190330611-9575a524-3176-492d-8fb6-d0dabdb42d59.jpg)

Only the file SafeExamBrowser.msi was given to us from an external supplier. But they unfortunately no longer suppy that file.

Thank you in advance for looking into it.

**Reply by dbuechel** [reply]
You most likely deleted the cached installer package (these are normally stored under `%SystemRoot%\Installer\` and should **never** be deleted manually). There is a utility from Microsoft which might help you resolve the issue: https://support.microsoft.com/en-us/topic/fix-problems-that-block-programs-from-being-installed-or-removed-cca7d1b6-65a9-3d98-426b-e9f927e1eb4d.

**Comment by hdenayer** [comment]
I had the same problem. Could it be that upgrading Windows 10 to 11 removed the installer package?

**Reply by dbuechel** [reply]
That is unlikely, but unfortunately not impossible, I'd reckon.
